README.md

```diff
@@ -1,127 +1,126 @@
 
 ```bash
-$ cd data
-$ wget --recursive --no-host-directories --cut-dirs 1 --reject index.html* --include-directories /data/batches/batch_uuml_thys_ver01/ http://chroniclingamerica.loc.gov/data/batches/batch_uuml_thys_ver01/
-$ cd ..
-$ docker exec -it open-oni /load_batch.sh batch_uuml_thys_ver01
+docker run -d \
+  -p 8983:8983 \
+  --name solr \
+  -v /$(pwd)/solr/schema.xml:/opt/solr/example/solr/collection1/conf/schema.xml \
+  -v /$(pwd)/solr/solrconfig.xml:/opt/solr/example/solr/collection1/conf/solrconfig.xml \
+  makuk66/docker-solr:4.10.4
 ```
 
-open-oni development
---------------------
+### Build open-oni
 
-First clone open-oni: `git clone git@github.com:open-oni/open-oni.git`. For a quickstart use `./dev.sh` (note for `docker-machine` use `./dev.sh $IP_ADDRESS_OF_MACHINE`), for more control use:
+Start the development open-oni.  This will install requirements if needed, and
+run various django admin commands as found in [`startup.sh`](startup.sh):
 
 ```bash
-$ docker build -t open-oni:dev -f Dockerfile-dev .
+mkdir -p data/batches data/cache data/bib
+
+docker run -i -t \
+  -p 80:80 \
+  --name open-oni-dev \
+  --link mysql:db \
+  --link solr:solr \
+  -v $(pwd)/open-oni:/opt/openoni \
+  -v $(pwd)/data:/opt/openoni/data \
+  open-oni:dev
 ```
 
-Migrate and start the development open-oni:
+In the above example the `open-oni` folder has been host volume mounted for
+dynamic development, while the `data` subfolder is coming from the current
+directory instead of the app directory. You can mount additional files /
+folders as needed. For example, to keep virtualenv files out of your source
+tree, you could add this:
 
-```bash
-$ docker run -i -t \
-$   -p 80:80 \
-$   --name open-oni-dev \
-$   --link mysql:db \
-$   --link solr:solr \
-$   -v /$(pwd)/open-oni/core:/opt/openoni/core \
-$   -v /$(pwd)/data:/opt/openoni/data \
-$   open-oni:dev
+```
+-v /tmp/CachedENV:/opt/openoni/ENV \
 ```
 
-In the above example the `core` folder has been host volume mounted for dynamic development. You can mount additional files / folders as needed. For example:
+Workflow
+---
 
-```
-$ -v /$(pwd)/open-oni/nebraska:/opt/openoni/nebraska \ # for a custom "app"
-$ -v /$(pwd)/settings.py:/opt/openoni/settings.py \ # note: DB_HOST and SOLR_HOST will be overrewritten
-```
+You should be able to develop in the `open-oni` repository folder as normal
+i.e. make feature branches etc.  When ready, make a pull request.  To test a
+pull request simply pull the remote feature branch (again, as normal) and run
+the container including the appropriate volume mount. Using Docker shouldn't
+change your regular development workflow much at all.
 
-Volume mounting is optional. You can always rebuild the image and rerun the container during development. To test modifications to the requirements file re-run the build command. Use the `dev.sh` script to simplify this.
+[`dev.sh`](dev.sh) can be used to stop, rebuild, and restart the dev container in a known
```

Comment:
I think this will fail if there is a mysql or solr container running, unless if the dev.sh file has changed since I was looking at it a few hours ago.  You have to stop and remove those containers first, otherwise.